pyproject.toml

```diff
@@ -27,4 +27,4 @@ use_parentheses = true
 skip_glob = ["build/*", "stubs/*"]
 # Don't split "import" and "from".
 force_sort_within_sections = true
-known_third_party = ["benchmark_dataset", "datasets", "golden_configs", "models", "numpy", "parameterized", "pytest", "recommonmark", "setuptools", "sklearn", "torch", "torchtext", "torchvision", "utils"]
```

Comment:
Does this fix the mypy issue that we keep having?

Reply:
no, it doesn't. I think the way pre-commit handles local tools is causing this for me. I am trying to get my local env to behave like the GitHub actions.